.semaphore/semaphore.yml

```diff
@@ -20,6 +20,11 @@ promotions:
         branch:
           - ^master$
 
+  - name: Stage PRs for review 
+    pipeline_file: pr-staging-deploy.yml
+    auto_promote:
+      when: "branch != 'master' or branch != 'release' or branch !~ '^dependabot\.'"
```

Comment:
This rule is needed so any PR automatically promoted except for `dependabot` PRs.